### PR TITLE
DOC: correct docstring on bluesky.utils.snake_cyclers

### PR DESCRIPTION
**Pull request by tacaswell**
Thanks to @cmazzoli for bring this to my attention.

## Description

Fixes a docstring to match what the code does.

## Motivation and Context

The docstring was wrong and confused @cmazzoli 

## How Has This Been Tested?

N/A

### CODE REVIEW

File: bluesky/utils/__init__.py [hunk 1]
```diff
@@ -592,11 +592,14 @@ def snake_cyclers(cyclers, snake_booleans):
     """
     Combine cyclers with a 'snaking' back-and-forth order.
 
+    If none of the cyclers are "snaked" this is the same as taking the product
+    (*****) of all the cyclers.
```
Comment (mrakitin) [inline bluesky/utils/__init__.py:596]: What does `(*****)` mean here?

**Review by mrakitin — COMMENTED** [review]
Looks good to me, thanks for fixing the misleading docstring, @tacaswell!

File: bluesky/utils/__init__.py (hunk 1 above)
Comment (tacaswell) [reply bluesky/utils/__init__.py:596]: That is attempting to make a bold `*` in rst.

**Comment by stan-dot** [comment]
this is now covered here https://github.com/bluesky/bluesky/pull/1776, 

closing

**Pull request closed without merging**